server/api/region.go

```diff
@@ -557,6 +558,43 @@ func (h *regionsHandler) GetTopSize(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+// @Tags region
+// @Summary Promote regions with the given key range.
+// @Param startkey query string true "Region Start key"
+// @Param endkey query string true "Region End key"
+// @Param limit query integer false "Limit count" default(16)
+// @Produce json
+// @Success 200 {object} string
+// @Failure 400 {string} string "The input is invalid."
+// @Router /regions/keys/promote [get]
+func (h *regionsHandler) PromoteRegionsInRange(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think `promote` is kind ambiguous, especially we already use the term for the meaning of _promote a learner peer to voter_.

Reply:
Could it be better to accept hex formatted keys?

Reply:
@disksing It's ok to me to change the API/function name here and do you know the API examples to know about the better format to accept hex formatted keys?

Reply:
I think maybe you can refer to `parseKey` in `region_command.go`?

Reply:
@disksing  Do you think we need to add a `format` parameter and support multi key format here?

Reply:
IMO, hex format is enough because it is our preferred format in the future. Of course, supporting more formats won't make it worse.

Reply:
Thanks for clarifying.